.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/local/bin/python3"
```

Comment:
I don't think this should be committed, it should be dev environment agnostic. Also, this assumes that everyone working on this (assuming they use vscode) have python installed in the same location. I personally use pipenv so I think this will mess up vscode for me haha